src/main/java/seedu/address/logic/crew/deletecrew/DeleteCrewCommand.java

```diff
@@ -24,18 +26,19 @@ public class DeleteCrewCommand implements Command {
 
     /**
      * Creates a command that, when executed, deletes the pilot with the
-     * given uuid.
+     * given id.
      *
      * @param uuid The index of the pilot to be deleted.
      */
-    public DeleteCrewCommand(String uuid) {
-        this.uuid = uuid;
+    public DeleteCrewCommand(String id) {
+        this.id = id;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        crewToDelete = model.getCrewManager().getItem(uuid);
-        model.deleteCrew(uuid);
-        return new CommandResult("Deleted crew: " + uuid);
+        int index = Integer.parseInt(id);
+        crewToDelete = model.getCrewManager().getItemByIndex(index);
```

Comment:
I think the `IndexOutOfBounds` exception is not properly handled here. May want to add exception handling logic.

Reply:
(Correct me if I am wrong)

Reply:
Done! More fundamental issues are raised in #110 #109